packages/scripts/src/admin-proposals/funding/0_Propose.ts

```diff
@@ -0,0 +1,141 @@
+// This script can be run against a mainnet fork by spinning a node in a separate terminal with:
+// HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
+// and then running this script with:
+// GCKMS_WALLET=<OPTIONAL-GCKMS-WALLET> \
+// RECIPIENT=<FUNDING-RECEIVER-ADDRESS> \
+// TOKEN=<OPTIONAL-TOKEN-ADDRESS> \
+// AMOUNT=<FUNDING-AMOUNT> \
+// PROPOSAL_URL=<PROPOSAL-URL> \
+// yarn hardhat run ./src/admin-proposals/funding/0_Propose.ts --network <network>
+
+import { strict as assert } from "assert";
+import { BigNumber, Signer } from "ethers";
+import hre from "hardhat";
+import { getGckmsSigner } from "@uma/common";
+import { getAddress, ERC20Ethers, ProposerV2Ethers, VotingTokenEthers } from "@uma/contracts-node";
+import { getContractInstance } from "../../utils/contracts";
+import { AdminProposalTransaction } from "../../upgrade-tests/voting2/migrationUtils";
+
+const { ethers } = hre;
+
+require("dotenv").config();
+
+const proposerAddress = "0x2bAaA41d155ad8a4126184950B31F50A1513cE25";
```

Comment:
interesting this is not an env

Reply:
agree, this is not optimal. better to use env when simulating and derive from keys when doing real proposal. Could improve this when refactoring getting proposer in a common lib for all admin proposals as per https://linear.app/uma/issue/UMA-1773/refactor-admin-proposal-scripts-to-share-common-functions